nativelib/src/main/scala/scala/scalanative/native/package.scala

```diff
@@ -118,4 +119,34 @@ package object native {
     @inline def toUInt: UInt     = toULong.toUInt
     @inline def toULong: ULong   = new ULong(value)
   }
+
+  /** Convert a CString to a java.lang.String */
+  def fromCString(cstr: CString)(implicit charset: java.nio.charset.Charset): String = {
+    val len = string.strlen (cstr).toInt
```

Comment:
The file doesn't parse with scalafmt's version of scala.meta, so scalafmt leaves the file unformatted.